src/CsvTextEditor/Commands/EditFindReplaceCommandContainer.cs

```diff
@@ -7,36 +7,24 @@
 
 namespace CsvTextEditor
 {
-    using Catel;
     using Catel.IoC;
     using Catel.MVVM;
-    using Models;
-    using Orc.CsvTextEditor.CsvTextEditorToolManagement;
+    using Orc.CsvTextEditor;
     using Orc.ProjectManagement;
 
     public class EditFindReplaceCommandContainer : EditProjectCommandContainerBase
     {
-        private readonly IServiceLocator _serviceLocator;
-        private ICsvTextEditorToolManager _csvEditorToolManager;
-
         #region Constructors
         public EditFindReplaceCommandContainer(ICommandManager commandManager, IProjectManager projectManager, IServiceLocator serviceLocator)
             : base(Commands.Edit.FindReplace, commandManager, projectManager, serviceLocator)
         {
-            Argument.IsNotNull(() => serviceLocator);
-
-            _serviceLocator = serviceLocator;
         }
         #endregion
 
         #region Methods
         protected override void Execute(object parameter)
         {
-            var activeProject = _projectManager.ActiveProject;
-
-            _csvEditorToolManager = _serviceLocator.ResolveType<ICsvTextEditorToolManager>(activeProject);
-
-            _csvEditorToolManager.AddTool<FindReplaceTextEditorTool>();
+            CsvTextEditorService.AddTool<FindReplaceTextEditorTool>();
```

Comment:
you want to add tool each time when you want to execute it?
looks strange
why not add tool somewhere in CsvTextEditor initialization or in CsvTextEditorService initialization?